rest_assured/testcases.py

```diff
@@ -257,6 +262,9 @@ def get_lookup_from_response(self, data):
         """
         return data.get(self.response_lookup_field)
 
+    def get_model(self):
+        return self.model_class if hasattr(self, 'model_class') else self.object.__class__
```

Comment:
Using `hasattr(self, 'model_class')` isn't correct here since the attribute exists and is `None`. Try following the usage of `getattr` in the code.

Also, I think it would help the users if you add an assertion here that if the new flag for not creating the object is `True` AND `model_class` doesn't return a class then raise an exception with a proper explanation.